src/v3/src/hooks/useInteractionCodeFlow.test.ts

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {
+  AuthSdkError, IdxStatus, IdxTransaction, TokenResponse,
+} from '@okta/okta-auth-js';
+import { waitFor } from '@testing-library/preact';
+import { renderHook } from '@testing-library/preact-hooks';
+import { getStubTransaction, getStubTransactionWithNextStep } from 'src/mocks/utils/utils';
+import { DescriptionElement, RedirectElement, WidgetProps } from 'src/types';
```

Comment:
does using absolute paths here not cause any issues?

Reply:
In tests they do not seem to cause a problem, however, in actual implementation files they definitely do 😅 

But I can update these for consistency 🙏